edxval/tests/data/wow.sjson

```diff
@@ -0,0 +1,5 @@
+{
+   "start": [10],
+   "end": [100],
+   "text": ["Hi, welcome to edxval."],
+}
```

Comment:
nit: New line

Reply:
this is a test data file -- not code file.

Reply:
I would still add a new line just for consistency.

Reply:
alright I'll add it